src/main.ts

```diff
@@ -24,6 +24,7 @@ async function run() {
         const status = core.getInput('status', { required: false });
         const whatsNewDir = core.getInput('whatsNewDirectory', { required: false });
         const mappingFile = core.getInput('mappingFile', { required: false });
+		const debugSymbols = core.getInput('debugSymbols', { required: false });
```

Comment:
The indentation here doesn't look right, I think it's using tabs instead of spaces?

Reply:
I'll fix that